shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/NotificationService.kt

```diff
@@ -0,0 +1,16 @@
+package org.jetbrains.kotlinconf
+
+import kotlinx.datetime.LocalDateTime
+
+interface NotificationService {
+    suspend fun requestPermission(): Boolean
+
+    fun post(
+        notificationId: String,
```

Comment:
It would be nice to use `value class NotificationId` here, similar to other places.

Reply:
We need to serialize it into an intent at some point on Android, and pass it to JS for interop, I don't want to complicate its type for now